bootstrap-scripts/bootstrap.sh

```diff
@@ -270,7 +270,7 @@ if [[ $use_system_ruby == "1" ]]; then
   [ -x "/usr/local/bin/bundle" ] || $USE_SUDO gem install -n /usr/local/bin bundler
   $USE_SUDO gem update -n /usr/local/bin --system
 
-else
+elif [[ "$CI" != 'true' ]]; then
```

Comment:
RVM not needed in CI thanks to the much faster `actions/setup-ruby` workflow.  Also, it's _very noisy_ with debugging and bumped up `bootstrap.sh` integration tests to ~30 minutes!